core/src/main/resources/lib/layout/breadcrumbBar.jelly

```diff
@@ -42,19 +42,10 @@ THE SOFTWARE.
         <ul id="breadcrumbs">
           <j:forEach var="anc" items="${request.ancestors}">
             <j:if test="${h.isModel(anc.object) and anc.prev.url!=anc.url}">
-              <li class="item">
-                <a href="${anc.url}/" class="${h.isModelWithContextMenu(anc.object)?'model-link':null}">
-                  <!-- Workaround to set desired name to first breadcrumb (instead of Jenkins) -->
-                  <j:choose>
-                    <j:when test="${anc.object == app}">
-                      ${%Dashboard}
-                    </j:when>
-                    <j:otherwise>
-                      ${anc.object.displayName}
-                    </j:otherwise>
-                  </j:choose>
-                </a>
-              </li>
+              <j:set var="mode" value="breadcrumbs" />
+              <l:breadcrumb title="${anc.object == app ? '%Dashboard' : anc.object.displayName}"
+                            href="${anc.url}"
+                            isMenu="${h.isModelWithContextMenu(anc.object)}" />
```

Comment:
Effectively breaks navigation if you are on a job, e.g. "Dashboard" is now bound to the URL you are at instead of root URL 👀 

(Untested)

Reply:
I reproduced with several pages of _Manage Jenkins_ (plugin manager etc.).

Reply:
Can't seem to reproduce there either, this time on Gitpod -

https://user-images.githubusercontent.com/43062514/178538693-896bc180-25a7-4ee0-ba74-45ac65d48041.mov

Reply:
https://user-images.githubusercontent.com/13383509/178586844-3538e8ac-c11c-4623-abc8-f1696a9b2690.mp4

🤔

Reply:
Would you mind trying again? I was missing a slash previously, not too sure if that would have fixed it.

Reply:
Might be context path dependent. I used the core-pr-tester, which doesn't have a context path.

Reply:
Awesome, happy to hear that.